packages/backend/src/persistance/messages/getIncomingMessages.ts

```diff
@@ -0,0 +1,24 @@
+import { Redis, RedisPrefix } from '../getDatabase';
+import { getMessages } from './getMessages';
+import * as Lib from 'dm3-lib/dist.backend';
+
+export function getIncomingMessages(redis: Redis) {
+    return async (
+        ensName: string,
+        limit: number,
+    ): Promise<Lib.messaging.EncryptionEnvelop[]> => {
+        const conversationIds = await redis.zRange(
+            RedisPrefix.IncomingConversations + ensName,
+            0,
```

Comment:
We're fetching just the first 100 conversations. I think that's sufficient. Otherwise we've to implement how we can implement pagination without knowing how much conversations we've in total